src/app/shared/components/map/map.component.ts

```diff
@@ -121,6 +131,8 @@ export class MapComponent implements AfterViewInit, OnDestroy {
         this.setMapLocation(L.latlng);
       }
     });
+
+    this.createUserRadius();
```

Comment:
this functionality should be added only for search-view case because otherwise this is applied also for create-address-form component. Move this method calling to 94-96 lines
![image](https://user-images.githubusercontent.com/27493959/196442169-ccbe3ef7-dcb1-4255-8c73-8411f76bf4d0.png)